### PR TITLE
Agent upgrade for team ownership

### PR DESCRIPTION
**Pull request by cbordier**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Update the upgrade procedure to take into account the team ownership

## Checklist for contributors

Before submitting this PR, please make sure:

* [X] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [X] You have signed the [Axway CLA](https://cla.axway.com/)
* [X] You have verified the technical accuracy of your change
* [X] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [X] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/connect_manage_environ/connected_agent_common_reference/upgrade_agent.md [hunk 1]
```diff
@@ -38,21 +38,51 @@ Correspond to this entry in environment variables: `MAIN_SUBSECTION_ENTRY=value`
 
 This [blog article](https://devblog.axway.com/dev-insights/amplify-central-agent-configuration-yaml-and-env-files/) contains another transformation sample related to the subscription section.
 
-### Axway gateway agents binary mode upgrade
+### Axway gateway agents upgrade
+
+For version 1.1.9 and later, the agent sets the owner team of an API Service based on the organization name the API belongs to. If an organization name matches a team name, the ownership of the API is assign to this team, and only the users belongings to this team can manage the APIs. It is no longer necessary to set `CENTRAL_TEAM` in the Discovery Agent variable and `owningTeam` in the Discovery Agent resource.
```
Comment (lbadenhop) [inline content/en/docs/connect_manage_environ/connected_agent_common_reference/upgrade_agent.md:43]: Is this last sentence correct?

**Comment by lbadenhop** [comment]
@cbordier Please verify my changes and see question.

File: content/en/docs/connect_manage_environ/connected_agent_common_reference/upgrade_agent.md (hunk 1 above)
Comment (cbordier) [reply content/en/docs/connect_manage_environ/connected_agent_common_reference/upgrade_agent.md:43]: @lbadenhop yes it is correct.
Default behavior is using the organization name to find a team set set the ownership. No agent configuration is require for that.

Comment (lbadenhop) [reply content/en/docs/connect_manage_environ/connected_agent_common_reference/upgrade_agent.md:43]: So there is a CENTRAL_TEAM variable in a Discover Agent variable? (variable within a variable?)

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**